internal/formats/common/spdxhelpers/to_syft_model.go

```diff
@@ -42,6 +49,35 @@ func ToSyftModel(doc *spdx.Document2_2) (*sbom.SBOM, error) {
 	return s, nil
 }
 
+// NOTE(jonas): this is a hack, SPDX doesn't seem to support a way
+// to describe what the SBOM is about: image, directory for example,
+// so I am trying to parse it from namespace. Syft generated SBOMs
+// have a type encoded in this field, so this approach works well
+// for SBOMs generated by Syft.
+func extractSourceFromNamespace(ns string) (src source.Metadata) {
```

Comment:
This func was added to move us closer to a functional decode/encode/decode cycle for SPDX. Without this info source.Metadata won't have a scheme and cannot be decoded.